lint/file.go

```diff
@@ -127,7 +127,7 @@ type enableDisableConfig struct {
 }
 
 func (f *File) disabledIntervals(rules []Rule) disabledIntervalsMap {
-	re := regexp.MustCompile(`^\s*revive:(enable|disable)(?:-(line|next-line))?(:|\s|$)`)
+	re := regexp.MustCompile(`^\s*revive:(enable|disable)(?:-(line|next-line))?(:)?([^\s]*)?(\s|$)`)
```

Comment:
This would also match `  revive:disable-next-line:`, which I suppose is okay. We should be able to handle the case when the rule name is empty.

Let me know if I'm missing something.

Reply:
Hi, yes the regexp matches `revive:disable-next-line:` and `revive:disable-next-line`. 

The modification handles the case of an empty rule name.
For example, with:

```go
// revive:disable-next-line
// The length of this comment line is over 80 characters, this is bad for readability.

func toto() {
	// revive:disable-next-line:
	fmt.Println("This line is way too long. In my opinion, it should be shortened.")
}
```
it works as expected.